app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Http\Requests\RegisterRequest;
+use Illuminate\Support\Facades\Storage;
+use App\Http\Resources\UserResource;
+use Validator;
+
+use App\Models\User;
```

Comment:
Mấy cái use đang liền mạch thì ko nên có dòng trống ở giữa chừng =))
```suggestion
use Validator;
use App\Models\User;
```